### ISSUE TITLE
Android Wear support

### ISSUE THREAD

**Issue by codeinabox** [cyclestreets/android#198]
Big fan of the app and use it quite regularly in London with my Bluetooth headphones. One feature I would love to see, and happy to try to create a PR for it, would be Android Wear support like Google Maps or Komoot where the watch shows how far away the next turn is.

**Comment by mvl22** [comment]
Yes, would be good to have this.

**Comment by jezhiggins** [comment]
Hi, 

Sorry for taking a while to response.  If you were able to raise a PR for Android Wear, that would be fantastic. Do feel free to contact me if you need assistance.

We do currently support Pebble, but I have absolutely no idea how relevant that is to Wear support.  It will indicate where you need to hook in, I guess.

**Comment by mvl22** [comment]
@codeinabox Yes, PRs would be very welcome. If you do have time to contribute, do let us know if you need any help getting the code running.

**Comment by codeinabox** [comment]
I've never developed anything for Android or Wear but once I have the experience I will try raising a PR.
